src/Components/NoteEdit/CreateNote.jsx

```diff
@@ -0,0 +1,187 @@
+import React from "react";
+import ReactQuill from "react-quill";
+import "./notecard.css";
+import "react-quill/dist/quill.snow.css";
+import { useNotes } from "../../Hooks/context/Notex-context";
+export const CreateNote = () => {
+  const { notes, setnotes, Notesdispatch, notesObj } = useNotes();
+
+  const formats = [
+    "header",
+    "bold",
+    "italic",
+    "underline",
+    "strike",
+    "list",
+    "bullet",
+    "indent",
+  ];
+  const ChangeHandler = (e) => {
+    setnotes({ ...notes, [e.target.name]: e.target.value });
+  };
+  const NotesubmitHandler = (e) => {
+    e.preventDefault();
+    Notesdispatch({ type: "ADD_NOTE", payload: { notes } });
+    setnotes('');
+  };
+  const modules = {
+    toolbar: [
+      ["bold", "italic", "underline", "strike"],
+      [{ header: 1 }, { header: 2 }],
+      [
+        { list: "ordered" },
+        { list: "bullet" },
+        { indent: "-1" },
+        { indent: "+1" },
+      ],
+      ["clean"],
+    ],
+  };
+
+  return (
+    <>
+      <form type='reset' onSubmit={(e) => NotesubmitHandler(e)}>
+        <div className="newNote-container">
+          <div className="input-container dis_flex">
+            <div className="title-container">
+              <p className="titel-label">
+                Title<small className="star_color">*</small>
+              </p>
+              <input
+                className="input-filed"
+                type="text"
+                name="title"
+                placeholder="title"
+                checked={notes.title}
+                onChange={(e) => ChangeHandler(e)}
+                required
+              />
+            </div>
+            <p className="titel-label">
+              Labels<small className="star_color">*</small>
+            </p>
+            <div className="label-container dis_flex">
+              <label htmlFor="home">
+                <input
+                  type="radio"
+                  name="label"
+                  id="home"
+                  value="home"
+                  checked={notes.label === "home"}
+                  onChange={(e) => ChangeHandler(e)}
+                  required
+                /> Home
+              </label>
```

Comment:
since all of these input data are recurring , you can keep it in an array and map it here, this will reduce your code